### ISSUE TITLE
If tests are executed in parallel with multiple runners (single feature->runner mapping) then only last run results will be shown.

### ISSUE THREAD

**Issue by prasadjshimpi** [qaprosoft/carina-cucumber#10]
If tests are executed in parallel with multiple runners (single feature->runner mapping) then only last run results will be shown.

the workaround did here to get this resolve 
Runner 1: 
        plugin = {"pretty"2
                "html:target/cucumber-core-test-report",
                "pretty:target/cucumber-core-test-report.txt",
                "json:target/cucumber-core-test-report.json",
                "junit:target/cucumber-core-test-report.xml"}
Runner 2: 
        plugin = {"pretty",
                "html:target/cucumber-core-test-**report1**",
                "pretty:target/cucumber-core-test-**report1**.txt",
                "json:target/cucumber-core-test-**report1**.json",
                "junit:target/cucumber-core-test-**report1**.xml"}
Runner 3: 
        plugin = {"pretty",
                "html:target/cucumber-core-test-**report2**",
                "pretty:target/cucumber-core-test-**report2**.txt",
                "json:target/cucumber-core-test-**report2**.json",
                "junit:target/cucumber-core-test-**report2**.xml"}

Actual Result: All runners result should be displayed
Expected result: Only last run results are getting displayed.

Result with a workaround: All runners results are getting displayed, but some runner results are shown in duplicate.

**Comment by SergeiZagriychuk** [comment]
Workaround in ticket's description is a valid flow which should be used and expected behavior.
You need to specify unique names for each of your runner.
Point with "some runner results are shown in duplicate" was not reproducible for me. Please create separate ticket providing more details and examples there if it's still reproducing for you.
@prasadjshimpi
